index.html

```diff
@@ -21,18 +23,198 @@
+                </li>
+                <li class="social-media-item"> 
+                    <a href=""><img src="assets/medium.svg" alt="Cristian's Medium" class="social-icon"></a>
+                </li>
+            </ul>
+            <!-- Resume button -->
+            <button type="button" class="resume-button">Get my resume</button>
+        </div>
+        <!-- Languages -->
+        <div class="skills-section">
+            <ul class="skills-container">
+                <li class="skills-title">Languages</li>
+                <li class="skills-image-container">
+                    <figcaption >
+                        <img src="assets/down-arrow.svg" alt="down-arrow-vector" class="skills-image">
+                    </figcaption>
+                </li>
+            </ul>
+            <ul class="skills-list">
+                <li class="skills-item">
+                    <div class="language-item">
+                        <figcaption class="language-image-container">
+                            <img class="language-image" src="./assets/js.png" alt="javascript">
+                        </figcaption>
+                        <h2 class="skills-item-title">JavaScript</h2>
+                    </div>
+                </li>
+                <li class="skills-item">
+                    <div class="language-item">
+                        <figcaption class="language-image-container">
+                            <img src="./assets/html.png" alt="html" class="language-image">
+                        </figcaption>
+                        <h2 class="skills-item-title">HTML</h2>
+                    </div>
+                </li>
+                <li class="skills-item">
+                    <div class="language-item">
+                        <figcaption class="language-image-container">
+                            <img src="assets/css.png" alt="css" class="language-image">
+                        </figcaption>
+                        <h2 class="skills-item-title">CSS</h2>
+                    </div>
+                </li>
+            </ul>
+            <!-- Frameworks -->
+            <ul class="skills-container active">
+                <li>
+                    <h2 class="skills-title">Frameworks</h2>
+                </li>
+                <li>
+                    <figcaption class="skills-image-container">
+                        <img src="assets/arrow.svg" alt="arrow-vector" class="skills-image">
+                    </figcaption>
+                </li>
+            </ul>
+            <!-- Skills -->
+            <ul class="skills-container active">
+                <li>
+                    <h2 class="skills-title">Skills</h2>
+                </li>
+                <li>
+                    <figcaption class="skills-image-container">
+                        <img src="assets/arrow.svg" alt="arrow-vector" class="skills-image">
+                    </figcaption>
+                </li>
+            </ul>    
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please use `ul` and `li` for categories in "About myself" section. i.e. `Languages`, `Frameworks`, and `Skills` should be list items.


Example
```HTML
<ul>
  <li>
    <h3>Languages</h3>
    . . . 
    <ul>
       <li>JavaScript</li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
     . . . 
  </li>
  <li> 
   <h3>Skills</h3>
    . . . 
  </li>
</ul>
```

Kindly note that the category items are wrapped in a single `ul` tag 👀🤗